flashinfer_bench/serve/low_level/errors.py

```
@@ -0,0 +1,60 @@
+
+        Returns
+        -------
+        None
+            This constructor initializes the exception state in place.
+        """
+        self.message = message
+        self.instruction_index = instruction_index
+        super().__init__(message)
+
+
+class TimeoutError(Exception):
```

Ruff (0.15.15)

[error] Variable `TimeoutError` is shadowing a Python builtin

(A001)